tests/integration/test_complete_run.cfg

```diff
@@ -85,7 +87,10 @@ years = "1850:1854:2", "1850:1854:4",
   [[ atm_monthly_180x360_aave ]]
   climo_diurnal_frequency = "diurnal_8xdaily"
   climo_diurnal_subsection = "atm_monthly_diurnal_8xdaily_180x360_aave"
+  environment_commands = "source /home/$USER/miniconda3/etc/profile.d/conda.sh; conda activate e3sm_diags_dev_20220614"
+  partition = compute
   sets = "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","enso_diags","qbo","diurnal_cycle","annual_cycle_zonal_mean","streamflow", "zonal_mean_2d_stratosphere",
+  walltime = "01:00:00"
```

Comment:
E3SM Diags now seems to take ~50 minutes rather than under 30 minutes.

Reply:
Longer run time is due to the additional ERA sets.